treeherder/push_health/tests.py

```diff
@@ -71,7 +72,7 @@ def get_history(failure_classification_id, push_date, num_days, option_map, repo
     else:
         previous_failures = json.loads(previous_failures_json)
 
-    return previous_failures, cache_key
```

Comment:
I was returning this `cache_key` for testing a long time ago, and forgot to remove it.  So it was useless.